src/libraries/Microsoft.Extensions.Configuration.FileExtensions/src/Resources/Strings.resx

```diff
@@ -118,9 +118,12 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="Error_ExpectedPhysicalPath" xml:space="preserve">
-    <value>The expected physical path was '{0}'.</value>
+    <value> The expected physical path was '{0}'.</value>
```

Comment:
was this diff intentional?

```suggestion
    <value>The expected physical path was '{0}'.</value>
```

Reply:
This was an intentional change, as this is [appended to the existing error message](https://github.com/dotnet/runtime/pull/51713/files#diff-20aa89fc83dc029b19defa0ad20c6d381d0920e0016762f18de42e162a014b3bR68). Is it better in this case to have another formatting to add the space between or what is suggested here?

Reply:
I didn't notice the intent. I guess it is fine as-is then.

cc: @safern in case he has other recommendations.